src/style.css

```diff
@@ -1 +1,90 @@
-/* Styles go here */
+* {
+  background-color: var(--white);
+  box-sizing: border-box;
+}
+
+:root {
+  --white: #fff;
+  --black: #000;
+  --blue: #00acdc;
+
+  --header-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 14px;
+  text-transform: uppercase;
+}
+
+.header {
+  box-shadow: var(--header-shadow);
+}
+
+.header-container {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: 0 auto;
+  max-width: 1200px;
```

Comment:
do not limit page width

Reply:
Yep, there is no sense to add min-width here. But it doesn't affect this task.